src/components/DateRangePickerInput.jsx

```diff
@@ -235,6 +236,8 @@ function DateRangePickerInput({
         regular={regular}
       />
 
+      {children}
```

Comment:
should this be checking if either one is focused? seems like this would result in the calendar always showing up, even when the fields are blurred.

Reply:
{children} here is the DayPicker, passed through all the way from DateRangePicker, where it is conditionally rendered based on `maybeRenderDayPickerWithPortal()`. This function renders null [here](https://github.com/GfxKai/react-dates/blob/master/src/components/DateRangePicker.jsx#L372-L374) if `isOpened()`, which checks whether one of the two inputs are focused, returns false. Hence, the focus check is done in the parent and we can just render unconditionally here and not worry 🎉